tokens/blocket.se/semantic.yml

```diff
@@ -169,12 +150,11 @@ s:
       inverted:
         _: white
         subtle: gray-50
-      link:
-        _: blue-600
-        hover: blue-600
-        active: blue-600
-        disabled: bluegray-300
+      link: blue-600
+      disabled: bluegray-300
       negative: red-600
       positive: green-600
 
     focused: aqua-400
+
+    divider: bluegray-200
```

Comment:
I didn't find this in the spreadsheet either but I suppose it was needed 😄

Reply:
It was from the other document. 😅  ...but the whole divider thing might change. But we'll keep this one for now. 🙂